ui/src/audioplayer/keyHandlers.js

```diff
@@ -25,7 +25,12 @@ const keyHandlers = (audioInstance, playerState) => {
     PREV_SONG: (e) => {
       if (!e.metaKey && prevSong()) audioInstance && audioInstance.playPrev()
     },
-
+    CURRENT_SONG: () => {
+      console.log(playerState)
+      if (playerState.queue[0]?.song?.albumId) {
+        window.location.href = `#/album/${playerState.queue[0].song.albumId}/show`
```

Comment:
This would go to the album of the first song of the queue, not the current playing song. Check the `prevSong` function above to see how to get the current song. You should also guard for when the queue is empty, to avoid a "index out-of-range"